packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -209,11 +209,22 @@ export default class Editor implements IEditor {
         scopeOrCallback: QueryScope | ((node: Node) => any) = QueryScope.Body,
         callback?: (node: Node) => any
     ) {
+        const result: HTMLElement[] = [];
         let scope = scopeOrCallback instanceof Function ? QueryScope.Body : scopeOrCallback;
         callback = scopeOrCallback instanceof Function ? scopeOrCallback : callback;
 
-        let range = scope == QueryScope.Body ? null : this.getSelectionRange();
-        return queryElements(this.core.contentDiv, selector, callback, scope, range);
+        let selectionEx = scope == QueryScope.Body ? null : this.getSelectionRangeEx();
+        if (selectionEx) {
+            selectionEx.ranges.forEach(range => {

```

Comment:
same here

Reply:
Here we are check whether the selection is null, if it is null the QueryScope will be Body type. 
I think we don't need range  if the Scope is body.